lib/scarlet/plugins/command.rb

```diff
@@ -140,18 +139,18 @@ def check_filters params
 
     # Checks whether the user actually has access to the command and can use it.
     # @param [Event] event The event that was recieved.
-    # @param [Symbol] privilege The privilege level required for the command.
+    # @param [Array<String>] groups The groups required for the command.
```

Comment:
Yeah, originally I planned to use Array of Strings, but I found using a proc more functional, no pun intended.